src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -439,36 +428,84 @@ def delete_cfs_configuration(cfs_config_name):
             logging.warning('Failed to delete CFS configuration "%s" '
                             'created by test: %s', cfs_config_name, err.stderr)
 
-    @staticmethod
-    def delete_all_cfs_configurations_matching_prefix(cfs_config_prefix):
+    @classmethod
```

Comment:
It still seems like it could be `staticmethod`. You would just have to change the call to `cls.delete_cfs_configuration` to use the class name.